db2i_misc.cc

```diff
@@ -100,7 +100,7 @@ EXTERN bool convertMySQLNameToDB2Name(const char* input,
   
   if (delimit)
     output[o++] = '"';
-  output[min(o, outlen-1)] = 0; // This isn't the most user-friendly way to handle overflows,
+  output[std::min( static_cast<size_t>(o), outlen-1)] = 0; // This isn't the most user-friendly way to handle overflows,
```

Comment:
I would probably change `uint o` to `size_t o` instead of the static_cast.

Reply:
Changed this to `size_t o`

https://github.com/zendtech/db2iengine/pull/23/files#diff-aa454e414a26f204ea0e1edcc31dbc9714afe333766b3d2388e6af82a41e01f0R85